README.md

```diff
@@ -10,4 +10,4 @@ juju deploy sdcore-smf --trust --channel=edge
 
 ## Image
 
-- **smf**: omecproject/5gc-smf:master-4ad802a
+- **smf**: `omecproject/5gc-smf:master-4ad802a`
```

Comment:
Should we target sdcore 1.3? https://docs.sd-core.opennetworking.org/master/release/1.3.html

Reply:
Yes, I think so. Done.